src/crc/_crc.py

```diff
@@ -10,36 +11,44 @@
 import sys
 from dataclasses import dataclass
 from typing import (
-    Any,
     BinaryIO,
-    ByteString,
+    Final,
     Iterable,
     Iterator,
-    List,
-    Optional,
     Union,
 )
 
 __author__ = "Nicola Coretti"
```

Comment:
```suggestion
__authors__ = ["Nicola Coretti", "Gert van Dijk"]
```

Reply:
I also plan to add a contributors page to the docs and add you if that's fine with you.

Reply:
I think the `__author__` remains singular even if it's a list of authors. Well, in fact there seems to be no standard on this, but this flake8 plugin says so. 😄  https://pypi.org/project/flake8-author/

A contributors page is cool, appreciated.